bugbear.py

```diff
@@ -174,13 +174,24 @@ def visit_ExceptHandler(self, node):
                 #    (MyError, MyError)  # duplicate names
                 #    (MyError, BaseException)  # everything derives from the Base
                 #    (Exception, TypeError)  # builtins where one subclasses another
+                #    (IOError, OSError)  # IOError is an alias of OSError since Python3.3
                 # but note that other cases are impractical to hande from the AST.
                 # We expect this is mostly useful for users who do not have the
                 # builtin exception hierarchy memorised, and include a 'shadowed'
                 # subtype without realising that it's redundant.
                 good = sorted(set(names), key=names.index)
                 if "BaseException" in good:
                     good = ["BaseException"]
+                # Find and remove aliases exceptions and only leave the primary alone
+                primaries = [
+                    primary
+                    for primary in B014.exception_aliases.keys()
+                    if primary in good
+                ]
+                for primary in primaries:
+                    aliases = B014.exception_aliases[primary]
+                    good = [e for e in good if e not in aliases]
```

Comment:
Perhaps this is a bit over-engineered. I could just define an edge-case for OSError. 

Thoughts?